examples/hosts/app-integration/external-data/README.md

```diff
@@ -6,15 +6,54 @@ Please note that the ideas explored here are experimental and under development.
 
 ## Scenario
 
-TBD
+This example demonstrates a scenario in which the Customers "source of truth" of their data lives in a service that is external to the Fluid service. The Customers can then:
+1. Import the external data into a Fluid collboration session.
+2. Export data from a Fluid collaboration session back to the source of truth.
+3. Sync updates between Fluid and the source of truth in as close to real-time as the scenario allows.
+
+In this case, the Fluid collaboration session serves as a "drafting surface" in which clients collaborate to create a draft of the data and then send the saved data back to the source of truth for long term storage.
 
 ## Strategy overview
 
-TBD
+Many services that would hold the "source of truth" data offer explicit commit style interfaces (e.g. vi REST call or similar) which are not well suited to rapid updates. However, they often expose third-party integration via REST APIS for querying and manipulating data, as well as webhooks for watching updates to the data.
+
+This repo contains an external service that serves as a mock external "source of truth" data server, that offers this REST API collection and webhook interfaces in ./src/mock-external-data-service. The APIs served by this external service are the following:
+
+1. POST `/register-for-webhook`
+2. GET `/fetch-tasks`
+3. POST `/set-tasks`
+4. POST `/debug-reset-task-list`
+
+Find the documentation for them in the code itself: (./src/mock-external-data-service/service.ts)[./src/mock-external-data-service/service.ts]
+
+Next we need a service that will register the webhooks and listen for incoming changes. In a true implementation, this registration would happen in the Fluid server-side, potentially in the Alfred service, as we have in our dev branch here: https://github.com/microsoft/FluidFramework/blob/dev/external-data-prototyping/server/routerlicious/packages/lambdas/src/alfred/index.ts#L463-L508.
+
+However, for the purposes of this example, we have built out a separate service to register for the webhook. It contains the following endpoints:
```

Comment:
I think this service exists even with the Alfred changes. I think the idea was that Alfred (or whatever realistic Fluid Service scenario we end up using) registers with the app's custom service, and not with the external data service directly.

Reply:
At minimum there's a translation layer - the external data is in some format that needs to be translated into a thing that Fluid can understand (e.g. turn some HTTP response into a Signal).  Also auth considerations - the Fluid service shouldn't pick up a responsibility of authenticating to arbitrary external data sources on our customers' behalf.

Later, for the bot pattern this customer service will also be where the bot runs most likely.

Reply:
Aha okay, I get it now. Will re-write given this new info!